### PR TITLE
Add Hurumap Tanzania  Primary School Teachers and Water Points stats

### PR DESCRIPTION
**Pull request by josephluvanda**
## Description

- Adds Primary school teachers dataset
- Add Water points  status dataset
- Updates google analytics tracking id 

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Screenshots
screenshot-tanzania hurumap org-2017-11-30-16-20-20-755

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: hurumap_tz/settings.py [hunk 1]
```diff
@@ -149,5 +148,7 @@
 
 LOGGING['loggers']['hurumap_tz'] = {'level': 'DEBUG' if DEBUG else 'INFO'}
 
+HURUMAP['na_label'] = ''
```
Comment (DavidLemayian) [inline hurumap_tz/settings.py]: What's the design decision here? Why have it blank instead of the default `na_label`?

Comment (josephluvanda) [reply hurumap_tz/settings.py]: There are datasets missing data points in districts and wards levels using N/A's make long pages

**Comment by DavidLemayian** [comment]
@darkdant3 Just waiting on the two other tasks on checklists to be completed for merge. ;)

**Comment by DavidLemayian** [comment]
@darkdant3 What happened?

**Comment by josephluvanda** [comment]
@DavidLemayian I'm not sure, I pulled and added documentation update.

**Comment by DavidLemayian** [comment]
@darkdant3 Force push perhaps?

**Comment by josephluvanda** [comment]
@DavidLemayian  Yes I force pushed

**Comment by DavidLemayian** [comment]
@darkdant3 Ok. Let's not force push to this repo please.

**Pull request closed without merging**